tests/test_datasets.py

```diff
@@ -48,3 +60,18 @@ def test_load_dataset(configuration: DatasetConfiguration, as_dataset: bool):
         }
     else:
         assert X_image.shape == (*configuration.image_size, configuration.n_features)
+
+
+def test_load_dataset_with_chunks():
+    """Test that the chunk size is respected when loading the as a dataset."""
```

Comment:
typo.  "loading this(?) as a dataset" or "loading the rasters(?) as a dataset"

Reply:
Thanks!